src/commands/api-types.d.ts

```diff
@@ -0,0 +1,224 @@
+  values: EnvVarValue[];
+  is_secret?: boolean;
+  updated_at?: string; 
+  updated_by?: UpdatedBy;
+}
+
+interface GetEnvParams {
+  accountId: string,
+  siteId?: string,
+  context?: Context,
+  scope?: EnvironmentVariableScope
+}
+
+interface DeleteEnvVarValueParams {
+  accountId: string,
+  key: string,
+  id?: string,
+  siteId?: string 
+}
+
+interface SetEnvVarValueBody {
+  context: string,
+  value: string,
+  contextParameter?: string,
+}
+
+interface SetEnvVarValueParams {
+  accountId: string,
+  key: string,
+  siteId?: string,
+  body: SetEnvVarValueBody
+}
+
+interface UpdateEnvVarBody {
+  key: string,
+  scopes: string[],
+  values: EnvVar[]
+  is_secret: boolean
+}
+
+interface UpdateEnvVarParams {
+  accountId: string,
+  key: string,
+  siteId?: string
+  body: EnvVar
+}
+
+interface createEnvVarParams {
+  accountId: string,
+  key?: string,
+  siteId?: string,
+  body: EnvVar[]
+}
+
+// Top-Level Interface
+interface SiteInfo {
+  id: string;
+  state: string;
+  plan: string;
+  name: string;
+  custom_domain: string | null;
+  domain_aliases: string[];
+  branch_deploy_custom_domain: string | null;
+  deploy_preview_custom_domain: string | null;
+  password: string | null;
+  notification_email: string | null;
+  url: string;
+  ssl_url: string;
+  admin_url: string;
+  screenshot_url: string;
+  created_at: string;
+  updated_at: string;
+  user_id: string;
+  session_id: string;
+  ssl: boolean;
+  force_ssl: boolean | null;
+  managed_dns: boolean;
+  deploy_url: string;
+  published_deploy: PublishedDeploy;
+  account_id: string;
+  account_name: string;
+  account_slug: string;
+  git_provider?: string;
+  deploy_hook: string;
+  capabilities: Capabilities;
+  processing_settings: ProcessingSettings;
+  build_settings: BuildSettings;
+  id_domain: string;
+  default_hooks_data?: DefaultHooksData;
+  build_image: string;
+  prerender: string | null;
+  functions_region: string;
+  feature_flags: FeatureFlags;
+}
+
+// Published Deploy Interface
+interface PublishedDeploy {
+  id: string;
+  site_id: string;
+  user_id: string;
+  build_id: string;
+  state: string;
+  name: string;
+  url: string;
+  ssl_url: string;
+  admin_url: string;
+  deploy_url: string;
+  deploy_ssl_url: string;
+  screenshot_url: string;
+  review_id: number | null;
+  draft: boolean;
+  required: string[];
+  required_functions: string[];
+  error_message: string;
+  branch: string;
+  commit_ref: string;
+  commit_url: string;
+  skipped: boolean | null;
+  created_at: string;
+  updated_at: string;
+  published_at: string;
+  title: string;
+  context: string;
+  locked: boolean | null;
+  review_url: string | null;
+  framework: string;
+  function_schedules: FunctionSchedule[] | [];
+}
+
+// Function Schedule Interface
+interface FunctionSchedule {
+  name: string;
+  cron: string;
+}
+
+// Capabilities Interface
+interface Capabilities {
+  [key: string]: Record<string, unknown>;
+}
+
+// Processing Settings Interface
```

Comment:
You don't need all these comments, they're all basically just saying the name of the interfaces are